packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -0,0 +1,246 @@
+<template>
+  <div>
+    <div v-if="!showEmptyState" class="flex flex-col gap-4">
+      <WorkspaceHeader :icon="Squares2X2Icon" :workspace-id="workspaceId" />
+      <div class="flex flex-col sm:flex-row gap-2 sm:items-center justify-between">
+        <div class="flex flex-col sm:flex-row gap-2">
+          <FormTextInput
+            v-model="search"
+            name="modelsearch"
+            :show-label="false"
+            placeholder="Search..."
+            :custom-icon="MagnifyingGlassIcon"
+            color="foundation"
+            wrapper-classes="grow md:grow-0 md:w-60"
+            :show-clear="!!search"
+            @change="updateSearchImmediately"
```

Comment:
Seems to be largely duplicated from ProjectList. If you make it so that projects are fed in through props (remember to define fragments), you can re-use that component instead of duplicating this new one

Reply:
Also: If you still end up needing a new component, don't do this debouncing stuff directly and use `useDebouncedTextInput`